user/metrics.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
Not sure if this should go in a separate package @smithwjv?

Reply:
Yeah stick it in a new one I reckon